### PR TITLE
Test against specific commit of OCCA

### PR DESCRIPTION
**Pull request by jeremylt**
This PR adds checking against a specific version of OCCA to check for failure due to changes in OCCA.

### CODE REVIEW

**Comment by jeremylt** [comment]
And we have the green checkmark again!

**Comment by jeremylt** [comment]
I think this is good solution. Thoughts?

screenshot from 2018-08-27 10-54-13

**Comment by jedbrown** [comment]
That's okay with me. We can collapse once the issue is fixed in Occa. Is there an issue in libocca/occa yet? https://github.com/libocca/occa/issues

**Comment by jeremylt** [comment]
There was an OCCA issue, but it got closed for some reason.

**Comment by jedbrown** [comment]
[The issue](https://github.com/libocca/occa/issues/198) so we don't lose track.

File: .travis.yml [hunk 1]
```diff
@@ -12,12 +12,22 @@ addons:
     - liblapack-dev
 
 env:
-  - FC=gfortran
+  - FC=gfortran OCCA_HEAD=HEAD
+  - FC=gfortran OCCA_HEAD=527494c
```
Comment (jedbrown) [inline .travis.yml]: Surely the second line overwrites the first.

Comment (jeremylt) [reply .travis.yml]: It is like the compiler specification; the build matrix is a combination of the different lines in runtime, environment, and exclusions/inclusions. So we get 2 compiler options x 2 env options = 4 runs

https://docs.travis-ci.com/user/customizing-the-build/#build-matrix

**Pull request merged**